components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -10267,25 +10262,21 @@ private JSONObject getSecurityAuditConfigurationProperties(String tenantDomain)
         List<APIResource> removedReusedResources = new ArrayList<>();
 
         for (URITemplate existingUriTemplate : existingUriTemplates) {
-
             // If existing URITemplate is used by any API Products
             if (!existingUriTemplate.retrieveUsedByProducts().isEmpty()) {
                 String existingVerb = existingUriTemplate.getHTTPVerb();
                 String existingPath = existingUriTemplate.getUriTemplate();
                 boolean isReusedResourceRemoved = true;
```

Comment:
Its better to keep additional new line to separate out conditional blocks of the code
```suggestion
                boolean isReusedResourceRemoved = true;

```

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/9604/commits/81fdfdcc985a2429bec9fcfa553704b98a3b6b87